### PR TITLE
S k song update tutorials and vstools

### PR DESCRIPTION
**Pull request by mijin85cho**
Additional update for tutorials and vstools

### CODE REVIEW

**Comment by mijin85cho** [comment]
Applied changes in #187 to close #180 and #187 .

File: docs/application/vstools/tools/project-wizard.md [hunk 1]
```diff
@@ -31,10 +31,12 @@ When you create a new project with a specific template, the Project Wizard autom
 To create a Tizen .NET application project:
 
 1.  To open the Project Wizard:
+
     a.  In the Visual Studio menu, go to **File &gt; New &gt; Project &gt; Templates &gt; Visual C\# &gt; Tizen**.
+
```
Comment (journey2w) [inline docs/application/vstools/tools/project-wizard.md]: Please change a, b, and c to 1, 2, and c. 
Markdown doesn't support type attribute.

File: docs/application/vstools/tools/project-wizard.md [hunk 2]
```diff
@@ -31,10 +31,12 @@ When you create a new project with a specific template, the Project Wizard autom
...
     b.  Select a template for your project.
 
-        The Project Wizard creates the project based on the selected template.
+    The Project Wizard creates the project based on the selected template.
```
Comment (journey2w) [inline docs/application/vstools/tools/project-wizard.md]: Please indent 3 spaces to align with 2nd ordered list.

File: docs/application/vstools/tools/project-wizard.md (hunk 1 above)
Comment (mijin85cho) [reply docs/application/vstools/tools/project-wizard.md]: Done.

File: docs/application/vstools/tools/project-wizard.md (hunk 2 above)
Comment (mijin85cho) [reply docs/application/vstools/tools/project-wizard.md]: Done.

**Comment by mijin80lee** [comment]
TD Sync Completed.(tutorials need to be checked later.)

**Pull request merged**